e2e/README.md

```diff
@@ -37,7 +37,7 @@ go build -o  $GOPATH/bin/newrelic-integration-e2e ./cmd/...
 
 You can now run the e2e tests locally
 ```shell
-LICENSE_KEY=${LICENSE_KEY} newrelic-integration-e2e --commit_sha=test-string --retry_attempts=5 --retry_seconds=60 \
+LICENSE_KEY=${LICENSE_KEY} EXCEPTIONS_SOURCE_FILE=${EXCEPTION_FILE} newrelic-integration-e2e --commit_sha=test-string --retry_attempts=5 --retry_seconds=60 \
```

Comment:
I would add few lines just to explain why we need that and how to select it

Reply:
done i added some extra explanation